services/director-v2/src/simcore_service_director_v2/utils/dask_client_utils.py

```diff
@@ -159,10 +160,14 @@ async def _connect_with_gateway_and_create_cluster(
         raise DaskGatewayServerError(endpoint=endpoint, error=exc) from exc
 
 
+def _is_internal_scheduler(authentication: ClusterAuthentication) -> bool:
```

Comment:
It is not clear to me what an "internal_scheduler" is, this implies the existence of outside_schedulers.

Does this refer to inside-simcore ("default"), or does it mean inside w.r.t. being on premise?

Reply:
The internal scheduler is the one defined within osparc-simcore deployment:
- dask-scheduler + N dask-sidecar(s)
i.e. what we currently have in all our deployments.

The external scheduler(s) are the one than can be defined by the users. They are indeed external to our stack and can be running from anywhere through the [osparc-dask-gateway](https://github.com/ITISFoundation/osparc-dask-gateway).
They can be on premise, running in some cloud provider, or just a raspberry...